sample-tracking-status-overview-app/src/main/groovy/life/qbic/portal/sampletracking/components/projectoverview/download/ManifestPresenter.groovy

```diff
@@ -24,21 +24,36 @@ class ManifestPresenter implements ComposeManifestOutput {
     }
 
     @Override
-    void readManifest(String printedManifest) {
+    void failedExecution(String reason) {
         try {
-            String newValue = Optional.ofNullable(printedManifest).orElse("")
-            viewModel.setGeneratedManifest(newValue)
+            sendFailure("Could not create download manifest. Reason: $reason")
         } catch (Exception e) {
             throw new OutputException(e.getMessage())
         }
     }
 
+    private void sendFailure(String message) {
+        notificationService.publishFailure(message)
+    }
+
+    /**
+     * To be called after successfully fetching sample codes with data for the provided code.
+     * @param projectCode the code of the project samples were counted for
```

Comment:
Major NIT: 
```suggestion
     * @param projectCode code of the project, for which the samples were counted for
```

Reply:
Thank you for your suggestion but I would prefer to leave it as is